### ISSUE TITLE
sessionStorage and tape-tests

### ISSUE THREAD

**Issue by Irazall** [unshiftio/sessionstorage#8]
Dear developers, 

I am using this library along with some unitests. Unfortunately it is not working correctly. This means, that I get an error that my sessionstorage is undefined.

Below I attach a sample of my code.
```
  sessionStorage.setItem('auth', JSON.stringify(mockAuthData));

  t.equal(
    sessionStorage.getItem('auth'),
    JSON.stringify(mockAuthData),
    'must have saved the authentication token and user to session storage'
  );
```

This certainly should be true, since I explicitly set the sessionstorage. But it is not! Interestingly it works with localstorage....
Could you please consider correcting this bug or tell me what can I do in order to test stuff that uses sessionstorage

Greetings

**Comment by lpinca** [comment]
This library only works in the browser. All the methods do [nothing](https://github.com/unshiftio/sessionstorage/blob/f7849ab1de520add24e2b41a0b8523f7bcffa39b/index.js#L16) when used in Node.js.

**Comment by Irazall** [comment]
Will this be implemented one day? This would be a really great feature!

**Comment by 3rd-Eden** [comment]
Feel free to contribute a fix in form of a PR. We would be happy to add the functionality to the library.

**Comment by 3rd-Eden** [comment]
Fixed in 0.1.0
